docs/interface.md

```diff
@@ -30,9 +30,9 @@ partial interface WorkerNavigator {
 };
 
 dictionary ShareData {
-  DOMString? title;
-  DOMString? text;
-  DOMString? url;
+  DOMString title;
+  DOMString text;
+  USVString url;
```

Comment:
@annevk knows the details, but basically anything that will end up serialized (encoded) and sent over the network benefits from using USVString, because then one can merely encode from Unicode code points, instead of first "decoding" the sequence of 16-bit values into Unicode.

Some things that aren't URLs also fit this description, for example the label argument here:
https://w3c.github.io/webrtc-pc/#peer-to-peer-data-api